src/sources/journald.rs

```diff
@@ -364,8 +371,16 @@ impl SourceConfig for JournaldConfig {
             .clone()
             .unwrap_or_else(|| JOURNALCTL.clone());
 
+        let systemd_version = get_systemd_version_from_journalctl(&journalctl_path).await?;
+
+        if !self.current_boot_only && (250..258).contains(&systemd_version) {
```

Comment:
```suggestion
        if !self.current_boot_only && (250..=257).contains(&systemd_version) {
```

Nit: although this is correct using `=257` reads a little better IMO